client/src/app/containers/Game/slice.ts

```diff
@@ -0,0 +1,122 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { Card, Game } from './models';
+
+export interface DealAction {
+    numShuffles: number;
+}
+
+export interface SelectCardsAction {
+    selectedCards: Card[];
+}
+
+export type CribAction = SelectCardsAction;
+export type PegAction = SelectCardsAction;
+
+export interface CutAction {
+    percCut: number;
+}
+
+export interface CountAction {
+    points: number;
+}
+export type CountHandAction = CountAction;
+export type CountCribAction = CountAction;
+
+export type GameAction =
```

Comment:
:book: maybe `InGameAction`? :shrug:

Reply:
Pros/cons? Why do you suggest a rename?

Reply:
:shrug: `GameAction` is just a weird name... as are all names...